app/code/Magento/Widget/Block/Adminhtml/Widget/Options.php

```diff
@@ -163,6 +163,8 @@ protected function _addField($parameter)
             }
         }
 
+        $data['value'] = html_entity_decode( $data['value'] );
```

Comment:
- Please use \Magento\Framework\Escaper::escapeHtmlAttr instead of native PHP library
- there is extra space after "(", please remove